oxidation/crates/parsec_api_crypto/Cargo.toml

```diff
@@ -18,7 +18,7 @@ libsodium-sys = { version = "0.2.7", optional = true }
 
 digest = { version = "0.10.1", optional = true }
 generic-array = { version = "0.14.4", optional = true, features = ["serde"] }
-sha2 = { version = "0.10.0", optional = true }
+sha2 = { version = "0.10.0", optional = true, features = ["asm"] }
```

Comment:
asm feature compatible with wasm32 ?

Reply:
Only x86, x86-64, and (partially) AArch64 architectures are currently supported.